### PR TITLE
Allow only supported Symfony versions

### PR DESCRIPTION
**Pull request by stephanvierkant**
See https://symfony.com/releases for supported versions.

### CODE REVIEW

**Review by Ciloe — APPROVED** [review]

**Comment by sanderdlm** [comment]
Is anything holding back this PR and https://github.com/friendsoftwig/twigcs/pull/190 from being merged? As far as I can see there are no significant changes in Symfony 6 that would break in this project, so would be nice to have support.

**Comment by OwlyCode** [comment]
The only thing holding this back is me being a bottleneck :)

I'll merge this when github actions have finished.

**Comment by bastien70** [comment]
Hello, maybe a new release since the PR is merged ?

**Pull request merged**
